### DISCUSSION TITLE
Transformers is useless

### DISCUSSION THREAD

**Discussion by jkaart** [TechReborn/TechReborn#2725] [Q&A]
What is the function of these? Looks like there is not any difference if you plug cable to low tier machine direct or use transformer between.
Multiple years ago IC2 (and GregTech also if I remember correct) machines blow up if you mix power tiers without transformer(s) and cables melt down if you run too much EU thru.

**Comment by Ayutac** [comment]
you can and should use them as intermediate blocks between cables, say all 5 or so, because cables have internal energy storage and flow back if you let them. transformers can be used as a one-way-only cable
